src/app/constants.js

```diff
@@ -1,98 +1,44 @@
-export const Component = {
-  DESCRIPTION: 'description',
-  STATUS: 'status',
-  GENES: 'genes',
-  CELL_SETS: 'cellSets',
-  SCATTERPLOT: 'scatterplot',
-  SPATIAL: 'spatial',
-  HEATMAP: 'heatmap',
-  LAYER_CONTROLLER: 'layerController',
-  CELL_SET_SIZES: 'cellSetSizes',
-  GENOMIC_PROFILES: 'genomicProfiles',
-  CELL_SET_EXPRESSION: 'cellSetExpression',
-  EXPRESSION_HISTOGRAM: 'expressionHistogram',
-};
+import {
+  Component as ComponentCurr,
+  DataType as DataTypeCurr,
+  FileType as FileTypeCurr,
+  CoordinationType as CoordinationTypeCurr,
+} from './constants-current';
+import {
+  Component as ComponentOld,
+  DataType as DataTypeOld,
+  FileType as FileTypeOld,
+  CoordinationType as CoordinationTypeOld,
+} from './constants-old';
 
-export const DataType = {
-  CELLS: 'cells',
-  CELL_SETS: 'cell-sets',
-  EXPRESSION_MATRIX: 'expression-matrix',
-  GENOMIC_PROFILES: 'genomic-profiles',
-  MOLECULES: 'molecules',
-  NEIGHBORHOODS: 'neighborhoods',
-  RASTER: 'raster',
-};
+function makeConstantWithDeprecationMessage(currObj, oldObj) {
```

Comment:
Can you add a code comment for what this does?  I wouldn't say it's immediately obvious.